src/components/atoms/Input/Input.model.ts

```diff
@@ -8,5 +8,6 @@ export interface IProps {
   className?: string;
   id?: string;
   ref?: HTMLInputElement;
-  handleOnChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onChangeFunc?: any; // do poprawienia typ
```

Comment:
We need only these props in input model

```javascript
export interface IProps extends React.InputHTMLAttributes<HTMLInputElement> {
  icon?: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
  fullWidth?: boolean;
  label?: string;
  error?: string;
}
```
and eventually your new props if this is required